src/main/resources/application.yml

```diff
@@ -69,6 +69,8 @@ spring:
         jdbc:
           lob:
             non_contextual_creation: true
+        hbm2ddl:
+          auto: update
```

Comment:
Added this so that Hibernate envers can generate the audit tables for testing locally. Just figuring out how to generate audit tables without impacting the actual tables.

Reply:
Sorted this put with a table name filter as discussed